x/feegrant/client/cli/query.go

```diff
@@ -84,7 +84,7 @@ $ %s query feegrant grant [granter] [grantee]
 // GetCmdQueryFeeGrantsByGrantee returns cmd to query for all grants for a grantee.
 func GetCmdQueryFeeGrantsByGrantee() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "grants [grantee]",
+		Use:   "grantee-grants [grantee]",
```

Comment:
How about the following 
```suggestion
		Use:   "grants-by-grantee [grantee]",
```